source/plugins/ruby/in_kube_podinventory.rb

```diff
@@ -230,6 +240,18 @@ def enumerate(podList = nil)
             telemetryProperties["SERVICE_ITEMS_CACHE_SIZE_KB"] = serviceItemsCacheSizeKB
             telemetryProperties["WINDOWS_CONTAINER_RECORDS_CACHE_SIZE_KB"] = @windowsContainerRecordsCacheSizeBytes / 1024
           end
+          if ExtensionUtils.isDataCollectionSettingsConfigured()
+            telemetryProperties["dataCollectionSettingsEnabled"] = "true"
+            if !@nameSpaces.nil? && !@nameSpaces.empty? && @nameSpaces.length > 0
+              telemetryProperties["DATA_COLLECTION_NAMESPACES"] = @nameSpaces
+            end
+            if !@nameSpaceFilteringMode.nil? && !@nameSpaceFilteringMode.empty?
+              telemetryProperties["DATA_COLLECTION_NAMESPACES_FILTERING_MODE"] = @nameSpaceFilteringMode
+            end
+            if @run_interval > 60
```

Comment:
can run_interval be less than the default(60 secs)

Reply:
> can run_interval be less than the default(60 secs)
> we dont support less than 60s. The allowed range is 1m(default) to 60m. If the customer provides any value outside of this range, then default (60s) will be used.